### ISSUE TITLE
How do I release a MC18 from a cradle, the "PersonalShopper" is not listed as a FEATURE_TYPE?

### ISSUE THREAD

**Issue by Cybrosys** [EMDK/xamarin#1]

**Comment by bhecox** [comment]
Hello,
To release the MC18 from the cradle you would use the Cradle API, and call unlock().

Have a look at the Personal Shopper API Guide under the Unlock header:
http://emdk.github.io/docs/3.1UP1/#guide-personalshopper

**Comment by Cybrosys** [comment]
I was hoping to be able to do it with the Xamarin Component but I guess I'll have to create a new jar binding library and map the personalshopper/cradle.

**Comment by bhecox** [comment]
Right, as of now, the Personal Shopper API's are not exposed in our Xamarin component, but will be in future releases.

**Comment by Cybrosys** [comment]
Do you know when there might be a release with the Personal Shopper APIs?
